client/src/pages/ModalBookingScreen.js

```diff
@@ -66,16 +78,31 @@ function ModalBookingScreen({ bookingDate }) {
 							</Form.Field>
 							<Divider inverted />
 							<label>Date: </label>
+<<<<<<< Updated upstream
```

Comment:
you have some problems with your git rebase/merge here - probably talk to a TA to help you resolve them?